src/shared/lambdaHandlerSearch.ts

```diff
@@ -11,7 +11,11 @@ export interface AbsoluteCharOffset {
 
 export type RangeOrCharOffset = Range | AbsoluteCharOffset
 
-export interface LambdaHandlerCandidate {
+export interface LambdaHandlerCandidate extends RootlessLambdaHandlerCandidate {
+    manifestUri: Uri
+}
+
+export interface RootlessLambdaHandlerCandidate {
     handlerName: string
     filename: string
     range: RangeOrCharOffset
```

Comment:
does the presence or absence of a couple fields justify a new type/interface?

Reply:
I'd consider it good for saving space in the code. There are a couple other options:

* Create the interface for `typescriptLambdaHandlerSearch.ts` (and deal with no interface for the other locations) since it's used 11 times in that file (I wouldn't consider it worth an extra 50 lines to remove the interface).
* Use `Pick<LambdaHandlerCandidate, ['handlerName', 'filename', 'range']>`. That way, we can ensure type safety when merging those fields into the `LambdaHandlerCandidate` interface later.

Reply:
oh hmm. If it saves code then great.